### PR TITLE
add cluster pane

### LINKED ISSUE #13: Add cluster component

### PR DESCRIPTION
**Pull request by flozender**
closes #13 
image

### CODE REVIEW

File: client/src/components/cluster.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+import { FaPlus } from "react-icons/fa";
+import CAvatar from "./clickableAvatar";
+
+const Cluster = ({ data, color }) => {
+  const follow = () => alert("Followed!");
+  const { name, users, defaultCluster } = data;
+  return (
+    <>
+      <Divider />
+      <Flex mt="2vh" mb="1vh" width="full" align="center" direction="row">
```
Comment (jcs98) [inline client/src/components/cluster.jsx:22]: Can you please add a horizontal scroll or left and right navigation arrows like in the [design](https://whimsical.com/fellowcomb-BFUk9i8u4e9jyLcuEivE4h) ?

File: client/src/components/cluster.jsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        mt="3vh"
+        mb="1vh"
+        mx="10"
+        px="8"
+        width="full"
+        align="center"
+        direction="row"
+      >
+        {users.map((user) => (
+          <CAvatar user={user} color="gray.400" size="md" />
```
Comment (jcs98) [inline client/src/components/cluster.jsx:51]: The CAvatar might take something like `color={user.color}` in the future.
Also if we don't plan on getting profile images from somewhere then we should probably display the users' names.

File: client/src/pages/home.jsx [hunk 3]
```diff
@@ -22,6 +23,23 @@ const Home = () => {
         }}
         color="orange"
       />
+      <Flex mt="5vh" mb="1.8vh" px="2vw" width="full">
+        <Heading width="full" size="lg">
+          Clusters
+        </Heading>
+      </Flex>
+      <Cluster
```
Comment (jcs98) [inline client/src/pages/home.jsx:31]: Should we put these in a loop?
Something like:
`[user.roles + user.follows].forEach(c => <Cluster ...c.data />)`
Or maybe we can do that once the api is ready

File: client/src/components/cluster.jsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+      <Divider />
+      <Flex mt="2vh" mb="1vh" width="full" align="center" direction="row">
+        <Text ml="8" fontSize="xl">
+          {name}
+        </Text>
+        <Spacer />
+        {defaultCluster ? (
+          ""
+        ) : (
+          <Button
```
Comment (jcs98) [inline client/src/components/cluster.jsx:30]: We'll need `follow`, `unfollow`, `join`, `leave` buttons- based on several conditions- let's discuss them

**Review by jcs98 — CHANGES_REQUESTED** [review]

File: client/src/components/cluster.jsx (hunk 2 above)
Comment (flozender) [reply client/src/components/cluster.jsx:51]: We're not going to have colors there, we'll have the GitHub pfp, so the color is just temporary

File: client/src/pages/home.jsx (hunk 3 above)
Comment (flozender) [reply client/src/pages/home.jsx:31]: Yep, let's do it when we have the API

File: client/src/components/cluster.jsx (hunk 4 above)
Comment (flozender) [reply client/src/components/cluster.jsx:30]: Yeah so the follow button will become unfollow if and when the user is already following it, we need to wait for the API for that. @utkarsh867 let us know when you're available.

**Pull request merged**